src/templates/home.html

```diff
@@ -50,8 +50,21 @@
                 
                 var strTaskList="";
                 for(var i=0; i < taskList.length;i++){
-                    strTaskList+="<a href='" + "{{url_for('task_view', taskID='')}}" + taskList[i].id + "' class='list-group-item'>";
+                    strTaskList+="<a href='" + "{{url_for('task_view', taskID='')}}" + taskList[i].id + "' class='list-group-item";
+                    var curTime = new Date(new Date().getTime() + new Date().getTimezoneOffset()*1000); // remove timezone
+                    var msTillDue = (new Date(taskList[i].duedate).getTime() - curTime) / 86400;
```

Comment:
Yes, we aren't supporting cross time zone right now.  Since this will be used at ISU and the Texas university I didn't think it was necessary for now since it was taking up so much time.